journal/modernized/14-chapter-11.adoc

```diff
@@ -590,9 +588,9 @@ These things, and the altogether appalling signs of the times,
 have combined to bring the minds of many into great humility and abasement of self.
 In addition to these outward circumstances,
 the general call from the ministry has been in effect,
-'how long will you see the standard,
+"`How long will you see the standard,
 and hear the sound of the trumpet? How long will you hear, and see,
-and disregard the loving-kindness of the Lord? It was rather remarkable,
+and disregard the loving-kindness of the Lord?`" It was rather remarkable,
```

Comment:
Are the quotes I changed in the correct placement from lines 591-593? I think I had trouble with them but I'm not sure why, I make notes for comments as I go, and I don't remember why I had trouble.